unit/solvers/smt2_incremental/smt2_incremental_decision_procedure.cpp

```diff
@@ -30,6 +30,8 @@
 #include <goto-symex/path_storage.h>
 #include <solvers/smt2_incremental/smt_to_smt2_string.h>
 
+#include "string_constant.h"
```

Comment:
I think the reason CI fails is this needs to be either `<util/string_constant.h>` or there must be a `string_constant.h` header somewhere else that hasn't been included with this PR.

Either way, an easy fix.

Reply:
Quite likely.